malboxes.py

```diff
@@ -269,10 +275,32 @@ def spin(parser, args):
     config['profile'] = args.profile
     config['name'] = args.name
 
-    f = open("Vagrantfile", 'w')
-    f.write(template.render(config))
+    with open("Vagrantfile", 'w') as f:
+        f.write(template.render(config))
     print("Vagrantfile generated. You can move it in your analysis directory "
-            "and issue a `vagrant up` to get started with your VM.")
+          "and issue a `vagrant up` to get started with your VM.")
+
+
+def append_to_script(filename, line):
+    """ Appends a line to a file."""
+    with open(filename, 'a') as f:
+        f.write(line)
+
+
+def add_to_user_scripts(profile):
```

Comment:
This is a side note, do not implement since this is a refactoring branch.

The source tree should not have user artifacts (or config) mixed with repository content. We need a least one (config.json right now) but I don't think we should add too many or maybe add a `config.d/` put the files there and have it in `.gitignore` and provide a `config.d.examples/`. Or maybe we should migrate `config.json` to `config.py` and have it be a simple dict with the values to generate the packer config, the ps1 user installer and everything else. For now leave this like that but let's think about it.